packages/transept/transept.0.1.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+version: "0.1.0"
```

Comment:
```suggestion
```

This is redundant, opam will override it with what infers from the opam file path in the repo